var/spack/repos/builtin/packages/bazel/package.py

```diff
@@ -17,6 +17,7 @@ class Bazel(Package):
     homepage = "https://www.bazel.io"
     url = "https://github.com/bazelbuild/bazel/releases/download/0.11.1/bazel-0.11.1-dist.zip"
 
+    version('0.19.0', 'dd5347f82482747a6d5cae6d4787b5da')
```

Comment:
the last time I tried updating bazel to 0.19.x I had problems with the system version of protobuf headers getting the build confused with the version that bazel vendors. Did you have any issues with protobuf when you installed bazel 0.19.x?

Reply:
FYI I'm getting this sort of error

```
  >> 158    ERROR: /tmp/bazel-0.19.2-entgmj6xx72ansnrmsdlpfqyuccil
            lrj/spack-src/src/main/cpp/util/BUILD:94:1: C++ compilation of rule
             '//src/main/cpp/util:port' failed (Exit 1): gcc failed: error exec
            uting command
```

Reply:
This is the symptom I was seeing for bazel. Have a look at the log file and look for the "SPACK_TARGET_ARGS must be set" line (or similar), it will most likely be truncated in the beginning ("mpiler not found" or something such)